app/views/editions/secondary_nav_tabs/edit/editable/components/_administration_availability.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= render "govuk_publishing_components/components/radio", {
+  heading: heading,
+  name: "edition[#{administration}_availability_attributes][authority_type]",
+  id_prefix: "conditional-checked",
+  items: [
+    {
+      value: "local_authority_service",
+      text: "Service available from local council",
+      checked: edition.editionable.send("#{administration}_availability").authority_type == "local_authority_service",
+    },
+    {
+      value: "devolved_administration_service",
+      text: "Service available from devolved administration (or a similar service is available)",
+      checked: edition.editionable.send("#{administration}_availability").authority_type == "devolved_administration_service",
+      conditional: render("govuk_publishing_components/components/input", {
+        label: {
+          text: "URL of the devolved administration website page",
+        },
+        name: "edition[#{administration}_availability_attributes][alternative_url]",
+        value: edition.editionable.send("#{administration}_availability").alternative_url,
```

Comment:
We should add an error message value here and an id that matches the `href` in the error message at the top of the page.